app/jobs/modsulator_job.rb

```diff
@@ -35,25 +35,23 @@ def perform(apo_id, uploaded_filename, output_directory, user_login, filetype, x
       if response_xml.nil?
         log.puts('argo.bulk_metadata.bulk_log_error_exception Got no response from server')
         log.puts("argo.bulk_metadata.bulk_log_job_complete #{Time.now.strftime(TIME_FORMAT)}")
+        return
       end
 
-      metadata_filename = generate_xml_filename(original_filename)
-      save_metadata_xml(response_xml, File.join(output_directory, metadata_filename), log)
+      metadata_path = File.join(output_directory, generate_xml_filename(original_filename))
+      save_metadata_xml(response_xml, metadata_path, log)
 
       if xml_only
         log.puts('argo.bulk_metadata.bulk_log_xml_only true')
       elsif filetype != 'xml' # If the submitted file is XML, we never want to load anything into DOR
-        # Load into DOR
         log.puts('argo.bulk_metadata.bulk_log_xml_only false')
-        update_metadata(apo_id, response_xml, log)
+        update_metadata(apo_id, response_xml, log) # Load into DOR
       end
 
-      finish_timestamp = Time.now.strftime(TIME_FORMAT)
-      log.puts("argo.bulk_metadata.bulk_log_job_complete #{finish_timestamp}")
+      log.puts("argo.bulk_metadata.bulk_log_job_complete #{Time.now.strftime(TIME_FORMAT)}")
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.